lib/Data.js

```diff
@@ -34,20 +34,31 @@ export class Suite {
         var summary = {
             passed: 0,
             failed: 0,
+            skipped: 0,
             runtime: 0
         };
 
         var allTests = this.getAllTests();
         allTests.forEach(function (test) {
-            if (test.status !== "passed") {
-                summary.failed++;
-            } else {
+            if (test.status === "passed") {
                 summary.passed++;
+            } else if (test.status === "skipped") {
+                summary.skipped++;
+            } else {
+                summary.failed++;
             }
             summary.runtime += test.runtime;
 
         });
 
+        if (summary.failed > 0) {
+            summary.status = "failed";
+        } else if (summary.passed > 0) {
+            summary.status = "passed";
+        } else {
+            summary.status = "skipped";
```

Comment:
I might have missed the proper discussion, but it is safe to say a test without failing or passing shall always be treated as "skipped"?

If this was discussed before, please ignore my comment.